test/integration/api-schema/download-json-schemas.sh

```diff
@@ -38,9 +43,14 @@ FILES=( \
   "user.json" \
 )
 
-mkdir -p ${schemadir}/errors ${schemadir}/transactions ${schemadir}/sourcemaps
+mkdir -p \
+  ${schemadir}/errors \
+  ${schemadir}/transactions \
+  ${schemadir}/spans \
+  ${schemadir}/metrics \
+  ${schemadir}/sourcemaps
 
 for i in "${FILES[@]}"; do
-  download_schema https://raw.githubusercontent.com/elastic/apm-server/master/docs/spec/${i} ${schemadir}/${i}
+  download_schema https://raw.githubusercontent.com/elastic/apm-server/v2/docs/spec/${i} ${schemadir}/${i}
```

Comment:
Side note: we should probably go through and leave some comments to update branch links before release.

Reply:
Good point. I have similar notes related to update documentation links in the [v2 tracking issue](https://github.com/elastic/apm-agent-nodejs/issues/471). I've added this as a task there as well